src/Factories/ChoiceFactory.php

```diff
@@ -65,12 +65,7 @@ public function getAddressType(): ?string {
 	}
 
 	public function getDefaultIncentives(): array {
-		if ( $this->featureToggle->featureIsActive( 'campaigns.membership_incentive.incentive' ) ) {
-			// TODO: This is currently hardcoded, we should make the list of incentives and defaults configurable
-			return [ 'tote_bag' ];
-		} elseif ( $this->featureToggle->featureIsActive( 'campaigns.membership_incentive.no_incentive' ) ) {
-			return [];
-		}
-		throw new UnknownChoiceDefinition( 'Address type configuration failure.' );
+		// TODO: This is currently hardcoded, we should make the list of incentives and defaults configurable
+		return [ 'tote_bag' ];
```

Comment:
Since the responsibility of `ChoiceFactory` is to create campaign-related instances/values, the workflow for removing campaigns is to replace all calls to the  `ChoiceFactory` method with with the static value/initialization and remove the  `ChoiceFactory` method.

Reply:
ok. I removed it, I hope I got you right